src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,128 @@
+              fill={isMobileMenuOpen ? '#2D3748' : '#F7FAFC'}
+            />
+          </svg>
+        </a>
+      </div>
+
+      <div className="group-right">
+        {/* Menu */}
+        <div className="desktop-menu-container">
+          <MenuList />
+        </div>
+        {/* Mobile Button */}
+        <div className="mobile-menu-btn">
+          <MobileMenuBtn onClick={() => setIsMobileMenuOpen(!isMobileMenuOpen)}>
+            {isMobileMenuOpen ? <X color={'#2D3748'} /> : <Menu color={'#F7FAFC'} />}
+          </MobileMenuBtn>
+        </div>
+      </div>
+
+      {isMobileMenuOpen && (
+        <div className="mobile-menu-container">
+          <MenuList isMobile />
+        </div>
+      )}
+    </NavContainer>
+  )
+}
+
+// STYLED COMPONENTES
+
+const NavContainer = styled.nav`
+  display: flex;
+  color: ${theme.colors.white};
+  .logo-container {
+    a {
+      display: inline-flex;
+    }
+  }
+  .group-right {
+    flex-grow: 1;
+    display: flex;
+    justify-content: flex-end;
+    .desktop-menu-container {
+      display: none;
+    }
+    .mobile-menu-btn {
+      display: flex;
+      align-items: center;
+    }
+  }
+  .mobile-menu-container {
+    width: 100%;
+    height: 100%;
+    padding-top: 1rem;
+  }
+  &.mobileMenuOpen {
+    position: fixed;
+    z-index: 999;
+    inset: 0;
+    padding-top: 1.5rem;
+    padding-bottom: 1.5rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+
+    background-color: ${theme.colors.white};
+    flex-wrap: wrap;
+  }
+  @media (min-width: 768px) {
+    .logo-container {
+    }
+    .group-right {
+      .desktop-menu-container {
+        display: flex;
+        align-items: center;
+      }
+      .mobile-menu-btn {
+        display: none;
+      }
+    }
+  }
+`
+
+const MobileMenuBtn = styled.button`
+  background-color: transparent;
+  display: flex;
+  padding: 0;
+  margin: 0;
+  border: 0;
+  &:focus,
+  &:active {
+    outline: none;
+  }
```

Comment:
Do we want to add some additional styling if we are removing the outline on focus? Keyboard users wont see any difference now

Reply:
100%, re-added the default focus ring for this solution.